src/crypto/bitcoin.ts

```diff
@@ -59,31 +60,27 @@ class Bitcoin implements ICrypto {
   // get address given an address type
   getAddress(derivationMode: string, xpub: string, account: number, index: number): string {
     switch (derivationMode) {
-      case this.DerivationMode.LEGACY:
+      case this.derivationMode.LEGACY:
         return this.getLegacyAddress(xpub, account, index);
-      case this.DerivationMode.SEGWIT:
+      case this.derivationMode.SEGWIT:
         return this.getSegWitAddress(xpub, account, index);
-      case this.DerivationMode.NATIVE:
+      case this.derivationMode.NATIVE_SEGWIT:
         return this.getNativeSegWitAddress(xpub, account, index);
       default:
         throw new Error(`Invalid derivation Mode: ${derivationMode}`);
     }
   }
 
   // infer address type from its syntax
-  //
-  // TODO: improve the prefix matching: make the expected prefix
-  // correspond to the actual type (currently, a `ltc1` prefix
-  // could match a native Bitcoin address type for instance)
   getDerivationMode(address: string) {
-    if (address.match('^(bc1|tb1|ltc1).*')) {
-      return this.DerivationMode.NATIVE;
+    if (address.match('^(bc1|tb1).*')) {
+      return this.derivationMode.NATIVE_SEGWIT;
```

Comment:
@hzheng-ledger I thought `ltc1` should be removed from Bitcoin and used in Litecoin instead. Can you please review this and confirm it's correct?